docker/nwchem/requirements.txt

```diff
@@ -0,0 +1,5 @@
+avogadro
+click
+jinja2
+openchemistry
+-e git+git://github.com/cclib/cclib.git#egg=cclib
```

Comment:
I don't think we are using cclib here?

Reply:
I think you are right. I have removed it.